src/pages/questionPage.js

```diff
@@ -32,12 +34,18 @@ export const initQuestionPage = () => {
 
   const currentQuestion = data.questions[data.currentQuestionIndex];
 
-  const questionElement = createQuestionElement(currentQuestion.text);
+  //Getting the question text
+  const questionElement = createQuestionElement(
+    quizData.currentQuestionIndex,
```

Comment:
I suggest using a parameter called data as this dataset is defined in the lines above and takes user sessions into account.
```suggestion
    data.currentQuestionIndex,
```